C++/Navio/Common/gpio.cpp

```diff
@@ -137,6 +136,22 @@ void Pin::toggle()
     write(!read());
 }
 
+uint32_t Pin::getPeriBase() const
+{
+    int rpi_version = getRaspberryPiVersion();
+
+    switch (rpi_version) {
+      case 1 ... 3:
+        return BCM2708_PERI_BASE;
+      case 4:
+        return BCM2711_PERI_BASE;
+      default:
+        std::cerr << "Unexpected version." << std::endl;
+        return false;
+    }
+}
```

Comment:
Move it to the next commit. This commit is just about to fix addresses.
And please(!) check the ret types again.

Reply:
Moved it, there shouldn't be ret problems now.